cmake/compile_tests/clang_omp.cpp

```diff
@@ -2,5 +2,8 @@
 
 int main(int argc, char** argv) {
   int thr = omp_get_num_threads();
-  return 0;
+  if ( thr > 0 )
+     return thr;
+  else
+     return 0;
```

Comment:
Shouldn't `omp_get_num_threads()` return a positive number on success?
Then,
```
return omp_get_num_threads()==0;
```
would at least return a success status.

Reply:
just fixing the unused variable by using it...this program isn't intended to run...just compile

Reply:
I know. You need to fix the indentation anyway (https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/Kokkos/detail/Kokkos/320/pipeline).